cypress.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "defaultCommandTimeout": 10000,
+  "video": false
```

Comment:
Disabling video as we dont publish the assets anywhere so its a waste. Also bumping the timeout as the execution time was inconsistent.